dev/is-changed.py

```diff
@@ -57,9 +57,15 @@ def main():
     changed_files = identify_changed_files_from_git_commits(
         "HEAD", target_ref=os.environ["APACHE_SPARK_REF"]
     )
-    changed_modules = determine_modules_to_test(
-        determine_modules_for_files(changed_files), deduplicated=False
-    )
+    if os.environ.get("APACHE_SPARK_REF", "") != "":
```

Comment:
Maybe we can just do:

```suggestion
    if os.environ.get("APACHE_SPARK_REF"):
```

Reply:
or `"APACHE_SPARK_REF" in os.environ`

Reply:
Sure.